test/framework/utils/constants.go

```diff
@@ -0,0 +1,5 @@
+package utils
+
+const (
+	COLORTELLER_IMAGE = "chinmay5j/colorteller:v1"
```

Comment:
ControllerName for the const name

Reply:
also why not use the tag latest?

Reply:
just to know exact version if anything changes on the image, so usually I don't use latest but in this case I can change it.